src/com/projectkorra/projectkorra/airbending/AirSuction.java

```diff
@@ -231,8 +232,8 @@ public void progress() {
 				if (Math.abs(entity.getVelocity().dot(push)) > knockback) {
 					push.normalize().add(entity.getVelocity()).multiply(knockback);
 				}
-
-				GeneralMethods.setVelocity(entity, push.normalize().multiply(knockback));
+				GeneralMethods.setVelocity(this,entity, push.normalize().multiply(knockback));

```

Comment:
Please change to `GeneralMethods.setVelocity(this, entity, push.normalize().multiply(knockback));`